packages/api/lambdas/sf-scheduler.js

```diff
@@ -41,11 +38,15 @@ async function handleScheduleEvent(event) {
   const customCumulusMeta = get(event, 'cumulus_meta', {});
   const payload = get(event, 'payload', {});
   const queueName = get(event, 'queueName', 'startSF');
-  const templateUri = get(event, 'template');
+  const messageTemplate = get(event, 'template');
+  const workflowDefinition = get(event, 'definition');
 
-  const messageTemplate = await getMessageFromTemplate(templateUri);
   const provider = await getProvider(providerId);
   const collection = await getCollection(collectionData);
+  const workflow = {
```

Comment:
Out of curiosity, what else does `workflowDefinition` contain?

Reply:
It will be the contents of the workflow file uploaded by kes, containing a `name`, `arn` and `definition` (the step function definition JSON.) These contents are the same as what is currently in each entry in `workflows/list.json` except `arn` is new and `template` has been removed.